docs/community/governance.md

```diff
@@ -90,36 +90,42 @@ One of their primary responsibilities is merging pull requests after all require
 
 For more information on Repository Maintainers--their responsibilities, who they are, and how one becomes a Maintainer--see the [README for Repository Maintainers][maintainers].
 
-## Area Experts
+## Working Groups (WGs)
 
-Area Experts are people with knowledge of specific components or technologies in the PowerShell domain. They are responsible for code reviews, issue triage, and providing their expertise to others.
+[Working Groups (WGs)][wg] are collections of contributors with knowledge of specific components or
+technologies in the PowerShell domain.
+They are responsible for issue triage/acceptance, code reviews, and providing their expertise to
+others in issues, PRs, and RFC discussions.
 
 They have [write access](https://help.github.com/articles/permission-levels-for-an-organization-repository/) to the PowerShell repository which gives them the power to:
 
 1. `git push` to all branches *except* `master`.
 1. Merge pull requests to all branches *except* `master` (though this should not be common given that [`master`is the only long-living branch](../git/README.md#understand-branches)).
 1. Assign labels, milestones, and people to [issues](https://guides.github.com/features/issues/).
 
-A list of Area Experts can be found [here][experts].
-
-### Area Expert Responsibilities
+### Working Group Responsibilities
 
-If you are an Area Expert, you are expected to be actively involved in any development, design, or contributions in your area of expertise.
+If you are a member of a Working Group, you are expected to be actively involved in any development, design, or contributions in the focus area of the WG.
+More information on the responsibilities of Working Groups can be found [here][wg],
+while current WG definitions and membership can be found [here][wg-definitions].
```

Comment:
Should we make clear that contributors don't have to be members of a WG to participate in a WG area, but WG members are empowered to make approval and rejection decisions